easybuild/framework/easyconfig/constants.py

```diff
@@ -51,4 +51,9 @@
     'OS_VERSION': (get_os_version(), "System version"),
     'SYS_PYTHON_VERSION': (platform.python_version(), "System Python version (platform.python_version())"),
     'SYSTEM': ({'name': 'system', 'version': 'system'}, "System toolchain"),
+
+    'OSPACKAGES_IBVERBS': (('libibverbs-dev', 'libibverbs-devel', 'rdma-core-devel'),
```

Comment:
To be in sync with `OS_PKG_OPENSSL_DEV` (see below), maybe make this `OS_PKG_IBVERBS_DEV`?

Reply:
Resovled in https://github.com/easybuilders/easybuild-framework/pull/3309/commits/2f967ae6496125390d1d94bda9ecba13cccd7563